src/components/mixins/DownloadMixin.vue

```diff
@@ -44,15 +44,15 @@ export default class DownloadMixin extends Vue {
         return this.$store.getters['profile/activeProfile'];
     }
 
-    async downloadCompletedCallback(downloadedMods: ThunderstoreCombo[]): Promise<void> {
+    async downloadCompletedCallback(downloadedMods: ThunderstoreCombo[], assignId: number): Promise<void> {
```

Comment:
Idea for another task: renaming the `assingId` to something more descriptive. If I read just this file "assignId" doesn't really tell me what this number is. `downloadId` or `progressId` oslt might be more helpful.

Reply:
Created a task for this.